.ci/jenkins/Jenkinsfile.buildchain

```diff
@@ -65,7 +65,13 @@ pipeline {
                         error(errorMessage)
                     }
                     println "[INFO] build-chain version recovered '${buildChainVersion}'"
-                    sh "npm install -g @kie/build-chain-action@${buildChainVersion} -registry=${NPM_REGISTRY_URL}"
+
+                    try {
+                        sh "npm install -g @kie/build-chain-action@${buildChainVersion} -registry=${NPM_REGISTRY_URL}"
```

Comment:
do you mean from a tagged source code, right? otherwise it would be dangerous to do that

Reply:
Yes, `kie/build-chain-action@${buildChainVersion}` should have a direct mapping on repo/branch (or tag)

Reply:
well, we are trying to cover a very special case here (to avoid the flow to fail in case the latest build-chain-action version is not available on the internal repository). I would keep it as simple as possible, just to install the library from the npm registry.

Reply:
Ok so my alternative proposal is to add a check (small Jenkins job?) on this repo that just verify that defined build-chain version is already available in our npm repository.
Currently the problem happened because we merged build-chain version bump while our internal npm proxy was not updated yet and we had no way to detect it until we saw jobs starting to fail.
In this way we preserve simplicity but we avoid to merge too soon

Reply:
I think it is a very good idea actually but the problem is already solved by this simple solution FMPOV. I mean in case the library is not available for the next hour or two after publishing it we will just use the latest installed one (because we catch the exception). Which is basically the same behavior you are basically proposing, "to keep using the latest available one until the new one is already sync". Having the job (your proposal) means something to implement and to maintain and we have additionally to wait to the job to be green and then wait to a person to check the PR and merge (probably after relaunching it few times).
With this current proposal, it is just to merge (the build-chain update PRs) and as soon the latest one is available it will be automatically installed for all.

I'm still thinking to keep this PR as the valid solution. wdyt?

Reply:
Isn't existing Jenkins job for this repo what we already need?
As far as I know it should fail already and we just need to check it before merging a bump. We can even configure a GitHub rule to make this Jenkins check mandatory.
I think it is ok to delay the merging of a version bump in this case.

Btw in general I would prefer a failure with proper error message instead of a silent fallback to a different version: the previous version could be not compatible in terms of configuration or even worst could have a different behaviour so a possible error will be hard to understand

Reply:
In summary if the Jenkins job is already reliable for this check my suggestion is keep this PR for the proper error message without the fallback and just wait to merge a version bump

Reply:
i will merge this change as it won't break anything